### PR TITLE
PowerShell Examples Resolved

### PR DESCRIPTION
**Pull request by ChrisKibble**
Fixed number of issues with formatting, updated some non-standard characters that weren't displaying correctly in CODE and may not when rendered in some web browsers, standardized on XYZ site code in two pages.

#MMS2019Docathon

### CODE REVIEW

**Comment by PRMerger14** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @ChrisKibble for the contribution! I'll review after the end of the contest.

#hold-off
(reviewers, please don't edit - I'll take care of it.)

File: sccm/mdt/troubleshooting-reference.md [hunk 1]
```diff
@@ -578,7 +578,7 @@ Figure 2. NTFS data streams
  Configure Unattend.xml to join a workgroup. Then, use the built\-in **Recover from Domain** task sequence step to add a step in the task sequence to join the domain after the static IP has been applied.  
 
 ### Driver Installation  
- To ensure the best possible user experience, installation of hardware devices and software drivers should run as seamlessly as possible, with little or no user intervention. Microsoft provides tools and guidelines to help create installation packages that meet this goal. For general information about driver installation, see [Device and Driver Installation](http://www.microsoft.com/whdc/driver/install/default.mspx).  
+ To ensure the best possible user experience, installation of hardware devices and software drivers should run as seamlessly as possible, with little or no user intervention. Microsoft provides tools and guidelines to help create installation packages that meet this goal. For general information about driver installation, see [Device and Driver Installation](https://msdn.microsoft.com/en-us/windows/desktop/dn653558).
```
Comment (aczechowski) [inline sccm/mdt/troubleshooting-reference.md]: Let's change this URL to a more modern version, I think this is it: https://docs.microsoft.com/windows-hardware/drivers/install/ (and make sure to not include the locale)

File: README.md [hunk 2]
```diff
@@ -4,7 +4,10 @@ This repository contains the conceptual technical content for [System Center Con
 
 We welcome and encourage community involvement as feedback (issues) and contributions (pull requests). For more information, see [How to use the Configuration Manager docs](https://docs.microsoft.com/sccm/core/understand/use-docs). 
 
+<!-- 
+(No longer exists - C. Kibble 2019-05-06)
```
[the remark this answers is not in the document]
Comment (ChrisKibble) [reply README.md]: Fixed.

**Comment by PRMerger8** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/develop/core/understand/windows-powershell-basics.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ System Center 2012 Configuration Manager SP1 introduced native PowerShell suppor
...
+ "Cmdlet" is also sometimes referred to as "Command-let".  The cmdlet is a basic instruction that you give Windows PowerShell.  All cmdlets are made up of two parts: a verb and a noun. They are separated by a hyphen â€˜-â€˜ character. Cmdlets are the building blocks of Windows PowerShell and are what do all the work for you.  Some cmdlets come with Windows, and others are installed with programs like Configuration Manager. An example of a Windows cmdlet that restarts a print job is: **Restart-PrintJob**. An example of a Configuration Manager cmdlet that retrieves a Configuration Manager site is: **Get-CMSite**.
```
[the remark this answers is not in the document]
Comment (ChrisKibble) [reply sccm/develop/core/understand/windows-powershell-basics.md]: I think I did this now the way you're suggesting, please let me know if I need to modify further.

**Comment by PRMerger12** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/mdt/troubleshooting-reference.md (hunk 1 above)
Comment (ChrisKibble) [reply sccm/mdt/troubleshooting-reference.md]: Link updated! :-)

**Comment by PRMerger13** [comment]
@ChrisKibble : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@aczechowski : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
I made one additional tweak, another set of those `â€˜` characters. Otherwise looks good!
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Comment by GitHubber17** [comment]
PR is merged.

**Pull request merged**